subsys/nrf_security/src/drivers/cracen/cracenpsa/src/key_management.c

```diff
@@ -533,6 +534,274 @@ static psa_status_t export_spake2p_public_key_from_keypair(const psa_key_attribu
 	}
 }
 
+static psa_status_t cracen_hkdf_sha256_hmac(const uint8_t *seed,
+					    const uint8_t *label,
+					    size_t label_len,
+					    uint8_t index,
+					    const uint8_t *opt_input,
+					    size_t opt_input_sz,
+					    uint8_t *out)
+{
+	cracen_mac_operation_t mac_op = {};
+	psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+	size_t length;
+	psa_key_attributes_t attr = PSA_KEY_ATTRIBUTES_INIT;
+
+	psa_set_key_type(&attr, PSA_KEY_TYPE_HMAC);
+	psa_set_key_bits(&attr, PSA_BYTES_TO_BITS(CRACEN_P256_KEY_SIZE));
+	psa_set_key_usage_flags(&attr, PSA_KEY_USAGE_SIGN_MESSAGE);
+
+	status = cracen_mac_sign_setup(&mac_op, &attr, seed, CRACEN_P256_KEY_SIZE,
+				       PSA_ALG_HMAC(PSA_ALG_SHA_256));
+	if (status) {
```

Comment:
```suggestion
	if (status != PSA_SUCCESS) {
```

For all of these to maintain consistency